src/Drupal/Driver/Cores/Drupal8.php

```diff
@@ -4,6 +4,7 @@
 
 use Drupal\Core\DrupalKernel;
 use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Routing\RouteObjectInterface;
```

Comment:
This has been introduced in Drupal 9.1, so adding this will break compatibility with Drupal 8.x and 9.0. I think it's OK to introduce this but we need to indicate this incompatibility in our `composer.json` file, using the `conflict` directive I mentioned in another comment.

Reply:
@AlexSkrypnyk @pfrenssen I've fixed this in https://github.com/jhedstrom/DrupalDriver/pull/253/files by hardcoding the constant value, so we keep compatibility with Drupal 8/9